src/main/scala/com/qubole/sparklens/app/ReporterApp.scala

```diff
@@ -76,6 +78,7 @@ object ReporterApp extends App {
     fs.open(path).readFully(byteArray)
 
     val json = (byteArray.map(_.toChar)).mkString
+    EmailReportHelper.generateReport(json, conf)
```

Comment:
I see three references to EmailReportHelper.generateReport. I was hoping we need to call this only once. What all scenarios are we supporting?